mix.exs

```diff
@@ -41,24 +41,25 @@ defmodule TwinklyMaha.MixProject do
   defp deps do
     [
       {:credo, "~> 1.4", only: [:dev, :test], runtime: false},
-      {:ecto, ">= 3.4.4 and < 3.4.7"},
+      {:ecto, ">= 3.4.4 and < 3.4.7", override: true},
```

Comment:
Why do we need to override ecto?

Reply:
because it was throwing an error compilation time, complaining it's not compatible with I think phoenix 1.5

Reply:
What if we upgrde ecto too? 🤔 , we should not be having a lot of broken implementation on the codebase

Reply:
Okay, I can try this